### PR TITLE
CGP-0042 - cBRL contract deploy

### PR DESCRIPTION
**Pull request by luisgj**
- init setup WIP-
- Adds DRAFT for cBRL CGP part 1

### CODE REVIEW

File: CGPs/cgp-0042.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# CGP [0042]: (Not a) Contract Release 6
+
+- Date: 2021-10-09
+- Author(s): @luisgj @volpe
```
Comment (martinvol) [inline CGPs/cgp-0042.md:4]:
```suggestion
- Author(s): @luisgj @martinvol
```

**Review by martinvol — APPROVED** [review]

File: CGPs/cgp-0042.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# CGP [0042]: cBRL contract deploy
```
Comment (YazzyYaz) [inline CGPs/cgp-0042.md:1]: please follow the new template preamble here: https://github.com/celo-org/governance/blob/main/CGPs/cgp-template.md

File: CGPs/cgp-0042.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+- Discussions To: [cBRL forum proposal](https://forum.celo.org/t/cbrl-stable-asset/1281)
+
+## Overview
+
+In this governance proposal, we propose to execute the Celo Core Contracts Upgrade (not a) Release 6, which mainly proposes the activation of the `StableTokenBRL` and `ExchangeBRL` smart contracts.
+
+TODO - Add link to release tag
+
+### Status
+- DRAFT = Feedback collection
```
Comment (YazzyYaz) [inline CGPs/cgp-0042.md:17]: Can delete this subsection

File: CGPs/cgp-0042.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+## Proposed Changes
+TODO - Gather Feedback on Proposed changes
```
Comment (YazzyYaz) [inline CGPs/cgp-0042.md:20]: Gathering feedback should be done outside the specifications of the governance proposal.

File: CGPs/cgp-0042.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+3. Deploy StableTokenBRL
+- Destination: Deploy StableTokenBRL implementation
+- Data:
+- Value: 0
+
+4. Deploy ExchangeBRL
+- Destination: Deploy ExchangeBRL implementation
+- Data:
+- Value: 0
+5. *TDB: Oracle setting, freezing the contracts, setting constitution parameters.*
```
Comment (YazzyYaz) [inline CGPs/cgp-0042.md:43]: add space after section 5.

**Review by YazzyYaz — CHANGES_REQUESTED** [review]

**Review by calicokittencat — APPROVED** [review]

**Comment by luisgj** [comment]
sorry mates. I was too slow to see the comments. Thanks for merging.

**Pull request merged**